src/libs/utils.js

```diff
@@ -89,7 +89,7 @@ export const hasAccessLevel = _.curry((required, current) => {
   return workspaceAccessLevels.indexOf(current) >= workspaceAccessLevels.indexOf(required)
 })
 
-export const canWrite = hasAccessLevel('WRITER')
+export const canWrite = bool => hasAccessLevel('WRITER')(bool)
```

Comment:
Should we have a `TODO` added to revert this change when we hopefully fix the type inference issue with `_.curry()`?

Reply:
Hmm... I'll add a TODO but the solution may be to keep this and add typing to the now-named parameter.